src/main/java/core/basesyntax/service/operation/impl/OperationReturnHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class OperationReturnHandler implements OperationHandler {
+
+    @Override
+    public void operationHandler(FruitTransaction fruitTransaction) {
```

Comment:
The method should be named `handleOperation` instead of `operationHandler` for better naming consistency.